packages/vc-verification/src/Ipfs-credential-resolver.ts

```diff
@@ -79,4 +98,43 @@ export class IpfsCredentialResolver implements CredentialResolver {
       .map(transformClaim)
       .filter(filterOutMaliciousClaims);
   }
+
+  async isCredential(credential: unknown): Promise<boolean> {
```

Comment:
IMO, using `VerifiableCredential` or `vc` is better. I think that an `offChainClaim` is technically a "credential" but it isn't a "verifiable" credential
```suggestion
  async isVerifiableCredential(vc: unknown): Promise<boolean> {
```